addons/purchase/purchase.py

```diff
@@ -1018,7 +1017,11 @@ def onchange_product_id(self, cr, uid, ids, pricelist_id, product_id, qty, uom_i
             # The 'or not uom_id' part of the above condition can be removed in master. See commit message of the rev. introducing this line.
             dummy, name = product_product.name_get(cr, uid, product_id, context=context_partner)[0]
             if product.description_purchase:
-                name += '\n' + product.description_purchase
+                desciption_purchase_label = ir_translation._get_source(
```

Comment:
OK, I get it. Please correct the variable name to `description_purchase_label` or even you can just call it `description`